specs/APUs-MOBILE/Phoenix.yaml

```diff
@@ -0,0 +1,32 @@
+name: Phoenix
+humanName: Phoenix
+type: APU Architecture
+topHeader: 'SELECT CPU:'
+data: 
+  Lithography: 5nm
+  Release Date: ''
```

Comment:
What's up with the empty release date?

Reply:
Ah I must've forgot to update this. 

In the inherit file I have a release date bit its not perfect, this was the best I could find: https://wccftech.com/amd-misses-ryzen-7040-phoenix-apu-launch-deadline-first-laptops-now-arriving-in-may/

I had put first of May 2023 but really it could be any day in May 2023

Reply:
I think you can just write `2023-05` and it should work okay